platforms/common/cortex/platform-common.c

```diff
@@ -119,6 +142,19 @@ void motor_drive(polar_t *command)
     int16_t right_command = (int16_t) (command->distance + command->angle);
     int16_t left_command = (int16_t) (command->distance - command->angle);
 
+    ctrl_t *ctrl = pf_get_ctrl();
+
+    if (ctrl_get_mode(ctrl) != CTRL_MODE_STOP) {
```

Comment:
and different of  CTRL_MODE_IDLE ?
and maybe CTRL_MODE_BLOCKED

Reply:
This test is to avoid log flood when mobile robot feedback loop is 'not-running'.
Goal is to keep users interactive session through UART usable feature.
CTRL_MODE_IDLE is currently dead code, but agree this should be added in case it appears in the object initialization implementation.
CTRL_MODE_BLOCKED, even if it's a transient mode, could produce data on wheels and robot movement which could be interesting, good to keep if can be probed.

Reply:
I agree